tests/test_sign_tx_cmd.py

```diff
@@ -0,0 +1,272 @@
+from ragger.navigator import NavInsID, NavIns
+from ragger.backend import RaisePolicy, SpeculosBackend
+from utils import ROOT_SCREENSHOT_PATH
+from vechain_client import VechainClient, Errors
+
+# Input
+# chaintag = 0xAA
+# expiration = 0x2D0
+# gaspricecoef = 128
+# gas = 21000
+# dependson = ""
+# nonce = "0x1234"
+# data = ""
+# to = "0xd6FdBEB6d0FBC690DaBD352cF93b2f8D782A46B5"
+# amount = 5
+# blockref = "0xabe47d18daa1301d"
+# transaction message
+transaction : bytes = bytes.fromhex("f83a81aa88aae47d18daa1301d8202d0e0df94d6fdbeb6d0fbc690dabd352cf93b2f8d782a46b5884563918244f4000080818082520880821234c080")
```

Comment:
could this be gerated using `vetBase.py` module? As done in `apdu_generator.py`?

Reply:
I also tried. But this module requires many outdated dependencies (which run only on old python version). I had first to create `apdu_generator.py` and then to have a dedicated environment where I patched some dependencies, use a dedicated python version, in order to be able to generate the apdus for tests inputs.

Reply:
sad but not a drama, there is no reason to change these transactions all the time. Maybe you can just add a comment at the top explaining that they were generating using `apdu_generator.py`?

Reply:
ok I will add a comment

Reply:
@tdejoigny-ledger @xchapron-ledger We have a new library can help https://github.com/vechain/thor-devkit.py#transaction